<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultDlg</class>
 <widget class="QDialog" name="ResultDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Result</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Time:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="lblElapsed">
       <property name="text">
        <string>2:18</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Poked-in:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLabel" name="lblTyped">
       <property name="text">
        <string>582</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Speed:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="lblSpeed">
            <property name="text">
             <string>253 зн/мин</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Errors:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="lblErrors">
            <property name="text">
             <string>4.6 %</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btnFinish">
       <property name="text">
        <string>Finish</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QPushButton" name="btnResume">
       <property name="text">
        <string>Resume</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnFinish</sender>
   <signal>clicked()</signal>
   <receiver>ResultDlg</receiver>
   <slot>finish()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnResume</sender>
   <signal>clicked()</signal>
   <receiver>ResultDlg</receiver>
   <slot>resume()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>finish()</slot>
  <slot>resume()</slot>
 </slots>
</ui>
